### ISSUE TITLE
Link reading/video material to assignments

### ISSUE THREAD

**Issue by ethanwhite** [datacarpentry/semester-biology#209]
It would be useful to link the reading/video material to the assignments as this would make it easier for self-guided students to find the material relevant to them. If this material was stored in a structured way it could also be used to auto-generate the schedule. `schedule.md` could basically be a YAML list of the assignments to be included and in what order and that would then auto-generate the page.

**Comment by ethanwhite** [comment]
If we go the auto-generated route then the `all_assignments` page could be made to look like:

```
Assignment 1: Reading, Exercises
Assignment 2: Reading, Exercises
...
```

With `Reading` and `Exercises` linked to the the relevant files.

**Comment by brymz** [comment]
This is addressed by #278
